WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -459,6 +471,10 @@ private extension SettingsViewController {
     func shouldShowPluginsSection() -> Bool {
         ServiceLocator.stores.sessionManager.defaultRoles.contains(.administrator)
     }
+
+    func shouldShowWhatsNew() -> Bool {
+        ServiceLocator.featureFlagService.isFeatureFlagEnabled(.whatsNewOnWooCommerce)
+    }
```

Comment:
I think we did decide to not show the row if there isn't any announcement to display right?

Reply:
You're right! I forgot this requirement. I've added a check to verify if there's an announcement to show.
Done! ✅